build-core-plugins.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd packages/kbn-types
```

Comment:
__optional nit:__ I believe you can also use `npm --prefix packages/kbn-types ...` and `npm --prefix "core_plugins/${i}" ...` to avoid doing `cd` several times.